examples/scanner.ts

```diff
@@ -17,12 +17,14 @@ import { platform } from 'os';
 
 import { scanner } from '../lib/';
 
-async function main() {
+const bootImageFolder = String(process.argv.slice(2));
+
+async function main(usbBootExtraFolder?: string) {
 	const adapters: scanner.adapters.Adapter[] = [
 		new scanner.adapters.BlockDeviceAdapter({
 			includeSystemDrives: () => true,
 		}),
-		new scanner.adapters.UsbbootDeviceAdapter(),
+		new scanner.adapters.UsbbootDeviceAdapter(bootImageFolder),
```

Comment:
nitpick: Do you also want to change this to `bootImageDir` so they all appear on grep?